modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/CommonAnalysisPlugin.java

```diff
@@ -419,7 +419,7 @@
                 throw new IllegalArgumentException(
                         "The [edgeNGram] token filter name was deprecated in 6.4 and cannot be used in new indices. "
                                 + "Please change the filter name to [edge_ngram] instead.");
-            } else if (version.onOrAfter(org.elasticsearch.Version.V_6_4_0)) {
+            } else if (version.onOrAfter(org.elasticsearch.Version.V_6_0_0)) {
```

Comment:
nit: you can probably remove the `if` here since all indices in v7 must have a version greater or equal to v6.0